src/linux_joystick.h

```diff
@@ -43,6 +43,7 @@ typedef struct _GLFWjoystickLinux
     int                     absMap[ABS_CNT];
     struct input_absinfo    absInfo[ABS_CNT];
     int                     hats[4][2];
+    struct ff_effect        *rumble;
```

Comment:
Had a look over the code and I don't see why this needs to be allocated with malloc. A pointer to the joystick is passed to `_glfwPlatformSetJoystickRumble` so the call to `ioctl` can just take the address of `linjs->rumble` and it removes some allocation code. Or am I missing something?